src/locales/cy/translation.json

```diff
@@ -514,6 +514,45 @@
         "sendTheCodeAgain": "Anfon y cod eto"
       }
     },
+    "passwordResetMfaSms": {
```

Comment:
Yes, the reasoning behind duplicating these translations was for each new page to have their own translation. For now the check-phone and check-email pages are the same for the log-in and reset-password journeys, but, we assumed they might differ later on since these new pages are before a user enters their password and the other pages appear after we validated the user's password.